### DISCUSSION TITLE
Commonsense Knowledge, Symbolic Knowledge Distillation paper: Why does the random predictor have such a high average precision?

### DISCUSSION THREAD

**Discussion by mayanks43** [eduardoblanco/csc696H-Spring24#10] [Q&A]
The authors ablate the critic model and also try a random predictor to critic the output of GPT-3. The random predictor has an average precision of 79.3. Is this expected? How do we justify such a high number?

**Comment by Urvika-gola** [comment]
I think its **not** expected that a 'random' predictor gives such a high accuracy. Maybe it could be due to the possibility of GPT3 having great quality output, or the criteria for judging the correctness of the outputs by the critic model is not too strict or well defined.

**Comment by eduardoblanco** [comment]
@mayanks43 Help me understand...

The test set is 50/50 correct/incorrect?

Random presumably means 50/50 correct/incorrect predictions? If so, I don't get how a random classifier gets 79% Precision.
